src/RaisedButton/RaisedButton.js

```diff
@@ -251,21 +251,22 @@ class RaisedButton extends Component {
 
   state = {
     hovered: false,
+    keyboardFocused: false,
     touched: false,
     initialZDepth: 0,
     zDepth: 0,
   };
 
   componentWillMount() {
-    const zDepth = this.props.disabled ? 0 : 1;
+    const zDepth = (this.props.disabled) ? 0 : 1;
```

Comment:
I'm wondering if there is an eslint rule for this. Well, as long as the eslint is green, that's fine for me.
However, it's even better to be consistent.